apps/src/StudioApp.js

```diff
@@ -1934,22 +1934,12 @@ StudioApp.prototype.handleEditCode_ = function (config) {
     }
   }
 
-  // Remove maker API blocks from palette and autocomplete, unless maker APIs are enabled.
-  // We didn't have access to project.useMakerAPIs() when dropletConfig
-  // was initialized, so include it initially, and conditionally remove it here.
+  // Remove maker API blocks from palette, unless maker APIs are enabled.
   if (!project.useMakerAPIs()) {
-    //// Remove maker blocks from the palette
+    // Remove maker blocks from the palette
```

Comment:
This code removes maker blocks from the toolbox configuration, which is separate from the droplet configuration.  The default App Lab toolbox [still contains maker blocks](https://github.com/code-dot-org/code-dot-org/blob/v2017-04-19.1/lib/cdo/shared_constants.rb#L191-L229), and any given level might also.

I haven't checked what happens if the toolbox configuration tries to reference a block that's present in the droplet configuration but not in any category.  If it's omitted without errors (and we consider that by design, not accidental) then I'd be comfortable removing this code too and leaning on the correct droplet config to "audit" the toolbox contents for us.